tear-down-azure.sh

```diff
@@ -1,13 +1,146 @@
 #!/bin/bash
```

Comment:
Just running this crashes in an interesting way. Also can't add -n as instructed

Reply:
What seems to be the issue?